src/main/java/pl/confitura/jelatyna/repository/ParticipantRepository.java

```diff
@@ -2,10 +2,29 @@
 
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.stereotype.Repository;
 import pl.confitura.jelatyna.domain.Participant;
 
 @Repository
 @RepositoryRestResource
 public interface ParticipantRepository extends CrudRepository<Participant, Long> {
```

Comment:
Tutaj samo repozytorium nie wystarczy. utworzenie uzytkownika to nie tylko jego zapisanie. potrzebujemy rowniez np wysłać do niego maila więc konieczny jest serwis.